server/api/lib/tags/index.js

```diff
@@ -0,0 +1,62 @@
+ const { tags, tagDataRules } = require('../../constants');
+ const PlanAreaChanges = require('../../../api/model/plan_area_changes');
+
+ const  generateTagsForPlan = async (planId) => {
+    let planTags = [];
+    let dataRules = [];
+    // HOUSING
+    const housingTag = tags['דיור'];
```

Comment:
Can you iterate on all the keys of the dictionary of tags instead of using just the housing key?
We want to make this function to work automatically on new tags that will arrive without adding code to this function. We would like this function to be 0 code maintenance for as long as we can.

I think that combining with moving the isTag function to individual files (a file per tag), you can move the function of "how to handle this dict key" to this file as well. (see the comment about moving isTag to seperate files below)